CreeDictionary/CreeDictionary/templates/CreeDictionary/word-entries.html

```diff
@@ -31,7 +31,15 @@ <h2 class="definition-title definition-title--search-result">
                                         data-cy="linguistic-breakdown">{% if result.linguistic_breakdown_head %}
                                     {{ result.linguistic_breakdown_head|join:" + " }}
                                     + {% endif %}
-                                    <strong>{{ result.lemma_wordform.text }}</strong> + {{ result.linguistic_breakdown_tail|join:" + " }}</span>
+                                    <strong>{{ result.lemma_wordform.text|add:":" }}</strong>
```

Comment:
Please add the colon to the template rather than using the `add` filter:

```suggestion
                                    <strong>{{ result.lemma_wordform.text }}:</strong>
```

The `add` filter [attempts to coerce both arguments to integers](https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#add), which is just weird...